packages/main/src/ToolbarSeparator.ts

```diff
@@ -20,7 +20,7 @@ import ToolbarItem from "./ToolbarItem.js";
  * @extends sap.ui.webc.main.ToolbarItem
  * @tagname ui5-toolbar-separator
  * @since 1.17.0
- * @abstract
+ * @implements sap.ui.webc.main.IToolbarItem
```

Comment:
I think we should keep @abstract since component doesn't have template tag

Reply:
My bad. Template is moved inside toolbarTemplate / toolbarPopoverTemplate functions instead of the decorator so you can ignore my previous comment.